src/context/loadCustomVolumeAndSnapshots.tsx

```diff
@@ -0,0 +1,28 @@
+import { fetchStorageVolume } from "api/storage-pools";
+import { fetchStorageVolumeSnapshots } from "api/volume-snapshots";
+import { LxdStorageVolume, LxdVolumeSnapshot } from "types/storage";
+import { splitVolumeSnapshotName } from "util/storageVolume";
+
+export const loadCustomVolumeAndSnapshots = async (args: {
```

Comment:
Not sure if we need this function, see below my comment on its usage.

Reply:
Yes, since the request for volume snapshots has been moved to its own component, this is no longer needed